.grunt-config/webpack.js

```diff
@@ -139,6 +141,7 @@ const externals = {
 	'@wordpress/i18n': 'wp.i18n',
 	react: 'React',
 	'react-dom': 'ReactDOM',
+	'web-vitals': 'WebVitals',
```

Comment:
TODO: `get import web-vitials working from node modules` 

cc: @joemcgill

Reply:
This should be working as of 3741e97fd